### ISSUE TITLE
ktest thinks he's compile

### ISSUE THREAD

**Issue by cos** [kframework/k-legacy#941]
```
$ ktest --help
Usage: kompile [options] <file>
  Options:
    --color
       Use colors in output. Default is on.
    --debug
       Print debugging output messages
       Default: false
```

**Comment by osa1** [comment]
Amazing. krun thinks that too.

The problem seems to be related with help messages -- they still work like how they work before.

**Comment by dwightguth** [comment]
Fixed #942
